stable/airflow/CHANGELOG.md

```diff
@@ -0,0 +1,197 @@
+# Changelog
```

Comment:
This is not really a changelog actually, this is an "upgrading guide", with what changed between 2 releases. Most of the time the minor changes / bug fixes are not described here, and I do not know if we want it.


I think it should be important to have clearly 2 types of informations:
- Changelog with the details of what changed (bug fixes....)
- upgrading information with only the changes that impact users (added/removed/moved/changed values in configuration,...)

My reference is the Home Assistant Release Blog, where they follow this structure at each major releases:
- human description of the purpose of the release
- upgrade guide
- breaking changes
- list of all changes (this section evolves with every minor releases they do)

Could be a good idea. Do you know if there are other charts that do this?